contentcuration/contentcuration/frontend/channelList/views/KCard/BaseCard.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+
+  <li
+    :class="['remove-list-style card',$computedClass(coreOutlineFocus)]"
+    tabindex="0"
+    data-focus="true"
+    @focus="cardFocus"
+    @hover="cardHover"
+  >
+    <component
+      :is="headerLevel"
+    >
+      <KTextTruncator
```

Comment:
[Required] The title text needs to be wrapped in `<router-link>` (you can see the reasons for it and the example of resulting markup in "Markup requirements" https://github.com/learningequality/kolibri-design-system/issues/529)